app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/TrackListViewModel.kt

```diff
@@ -62,6 +73,30 @@ constructor(
         }
   }
 
+  override fun addTrackToList(listName: ListType, track: Track) {
+    _uiState.value.tracks += track
```

Comment:
I think it's cleaner to edit `_uiState.value` by assigning a modified copy to it than to have `tracks` be a var, just like we did previously in other functions